LoopKit/GlucoseKit/GlucoseMath.swift

```diff
@@ -90,7 +90,8 @@ extension BidirectionalCollection where Element: GlucoseSampleValue, Index == In
     /// - Returns: An array of glucose effects
     func linearMomentumEffect(
         duration: TimeInterval = TimeInterval(minutes: 30),
-        delta: TimeInterval = TimeInterval(minutes: 5)
+        delta: TimeInterval = TimeInterval(minutes: 5),
+        velocityCap: HKQuantity = HKQuantity(unit: HKUnit.milligramsPerDeciliter.unitDivided(by: .minute()), doubleValue: 4.0)
```

Comment:
Is this going to be used by DIY? Also, `velocityCap` name is a bit misleading. Perhaps `velocityMaximum`? Also, this seems like something that deserves a comment as to *why* this is being pinned to a maximum velocity.

Reply:
Yes, it will be accepted by DYI.

I'll include update the variable name and include a comment to help explain the rationale.